src/domains/idpay-common/03_database.tf

```diff
@@ -75,4 +75,245 @@ module "mongdb_collection_onboarding_citizen" {
   ]
 
   lock_enable = false #TODO temporary only for dev
-}
\ No newline at end of file
+}
+
+# Collections
```

Comment:
Please use a variable for the lock_enable so we can avoid errors in future and have a separate configuration for each env

Reply:
the lock now can be put to true on each environment